web/themes/custom/bixal_uswds/src/sass/styles.scss

```diff
@@ -5,7 +5,8 @@
 
 // USWDS individual components
 // to include everything, just code it as @forward "uswds";
-@forward "uswds";
+@forward "uswds-core";
+@forward "usa-skipnav";
```

Comment:
Skipnav isn't displaying on focus, but we can target that in another issue if its a blocker. Seems to be also happening on production.

| Expected | Actual |
| :----- | :---- |
|<img src="https://github.com/user-attachments/assets/a8837576-367f-4e2b-8125-2e23df3fb620" width="200" /> | <img src="https://github.com/user-attachments/assets/7fbd00bc-8c75-41c3-be7c-96d79dc5c533" width="200" />|